DependencyInjection/Compiler/AddProcessorsPass.php

```diff
@@ -36,6 +37,14 @@ public function process(ContainerBuilder $container)
 
                 if (!empty($tag['handler'])) {
                     $definition = $container->findDefinition(sprintf('monolog.handler.%s', $tag['handler']));
+                    $parentDef = $definition;
+                    while (!$parentDef->getClass() && $parentDef instanceof ChildDefinition) {
+                        $parentDef = $container->findDefinition($parentDef->getParent());
+                    }
+                    $class = $container->getParameterBag()->resolveValue($parentDef->getClass());
+                    if (!method_exists($class, 'pushProcessor')) {
+                        continue;
```

Comment:
should this case silently ignore the push or fail ? It might be unexpected to explicitly reference an handler and then have the config ignored.

Reply:
Yeah, indeed. Fixed.